packages/wmr/src/lib/prerender.js

```diff
@@ -146,19 +120,44 @@ async function workerCode({ cwd, out, publicPath }) {
 				routes.push({ url, _discoveredBy: route });
 			}
 		}
-		const body = (result && result.html) || result;
+
+		/**
+		 * @type {{ lang?: string, title?: string, links?: any[], metas?: any[]}}
```

Comment:
Couldn't this just use HTML? Right now there's no way to inject a script or style element.

Reply:
Good point, haven't thought about style or script tags. How do you feel about an API like this:

```ts
interface HtmlHead {
  lang: string;
  title: string;
  // placeholder name, not sure what else to call it
  elements: string[]; // meta-/script-/style-tags pre-serialized to a string
}
```